src/util/daos_vol_err.c

```diff
@@ -171,12 +171,12 @@ H5_daos_err_to_string(int ret)
             return "service group not attached (DER_NOTATTACH)";
         case -DER_MISMATCH:
             return "version mismatch (DER_MISMATCH)";
-#ifdef DER_EVICTED
-        case -DER_EVICTED:
-            return "rank has been evicted (DER_EVICTED)";
-#else
+#if DAOS_API_VERSION_MAJOR > 1 || DAOS_API_VERSION_MINOR > 2
         case -DER_EXCLUDED:
```

Comment:
I'm confused by the `||`, should not that be `&&` ? it sound like this would not work with v 0.3.0 for example (not that it's likely to happen)

Reply:
but you have a point for older version, like if we had major version of 0 (which we don't) then if we had a minor version 3 or more, this would break

Reply:
so this can be 
if (major > 1) || (major == 1 && minor > 2)
but since we know daos with major 0 (or undefined) is not supported and probably current vol doesn't even compile with that, i think this is not needed.

Reply:
ok :) sure no that's fine. I think we should just create a macro at some point like DAOS_MIN_REQ(major,minor) or something similar, just to keeps things simple.